File: backend/src/main/java/ru/focsit/backend/dto/UserDto.java
package ru.focsit.backend.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UserDto {
    private Long userId;

    private String username;

    private String userEmail;

    private String password;

    private Long userCountryId;

    private Long userRoleId;
}

File: mobile/app/src/main/java/ru/focsit/mobile/data/UserDto.kt
package ru.focsit.mobile.data

/**
 * Данные пользователя системы.
 * Используется для представления информации о пользователе, такой как его ID, имя пользователя, email и роль.
 *
 * @property id Уникальный идентификатор пользователя.
 * @property username Имя пользователя.
 * @property email Адрес электронной почты пользователя.
 * @property role Роль пользователя в системе (например, "админ", "модератор", "пользователь").
 */
data class UserDto(
    val id: Long,
    val username: String,
    val email: String,
    val role: String
)